### ISSUE TITLE
Better the tooltips UI/UX in Results graph

### ISSUE THREAD

**Issue by ClemenceK** [Boavizta/e-footprint-interface#110]
On Results graphs

On tooltips (bubble on hover with details of results) on both results curves:
 
# Make "tooltip"  more usable: 
- Contrast should be higher for accessibility
- Fontsize should be bigger
- Tooltip should not move wildly — make it slower or fix position

# Make tooltips more informative:
on top of teh content of tooltip:
- Add sum (should equal sum of elements below) Rationale: today the only way to know total figure at given time is summing in one's head; or reading the curve, approximately; sum would allow clear reading of useful figures)
- Add units to sum
- example: 
Jan 2028
0.4 t CO2-eq 
_then details as today (image below)_

<img width="391" alt="Image" src="https://github.com/user-attachments/assets/56380dfc-bb54-4730-92d2-062e5fd28b65" />
